CHANGELOG.md

```diff
@@ -11,6 +11,8 @@ Versions
 * Python SRC Config : Add linux/amd64 and linux/arm64 as values for the new config property base_platforms
 * CI : use docker/setup-qemu-action@v2 and docker/setup-buildx-action@v1 as gitflows step actions
 * Images : Only [OCI images](https://github.com/opencontainers/image-spec/blob/main/spec.md) for amd64 architecture are built
+* Bitcoind : Updating to bitcoind to 23.0
```

Comment:
Should be under `2022-08-31` :)

Reply:
Done !
Commit id : 6fd876e2eaf7b142d0dd0f74b326e9c5b11f8cd6